### PR TITLE
LVNOD-39 Create route rendering

### PR DESCRIPTION
**Pull request by avoytovich**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [x] @denisbodnar
- [ ] @yuliannakorol
- [x] @AlbaMontis 

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/src/components/Address/Address.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router';
 
 export default class Address extends React.Component {
   render() {
-    return <h2>You can find us in Lviv</h2>;
+    return (
+    	<div>
+    		<h2>You can find us in Lviv</h2>
+    		<Link to="/contacts/address/another_region">another_region</Link>
```
Comment (denisbodnar) [inline app/src/components/Address/Address.js]: Use spaces, not tabs

File: app/src/components/Another_region/Another_region.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class Another_region extends React.Component {
+  render() {
+    return <h1>You can find us in Another_region</h1>;
+  }
+}
```
Comment (denisbodnar) [inline app/src/components/Another_region/Another_region.js]: Add line

File: app/src/components/index.js [hunk 3]
```diff
@@ -5,3 +5,4 @@ export Contacts from './Contacts/Contacts';
 export Home from './Home/Home';
 export Main from './Main/Main';
 export Maps from './Maps/Maps';
+export Another_region from './Another_region/Another_region';
```
Comment (denisbodnar) [inline app/src/components/index.js]: Add a new line

File: app/src/router.js [hunk 4]
```diff
@@ -1,12 +1,17 @@
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
-import { About, Home, Contacts, Main } from './components';
+import { Route, IndexRoute, Link } from 'react-router';
```
Comment (denisbodnar) [inline app/src/router.js]: Imported Link but didn`t use

File: app/src/router.js [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
+import { About, Address, Another_region, Home, Contacts, Main, Maps } from './components';
 
 const Routes = (
   <Route path='/' component={Main}>
     <IndexRoute component={Home}/>
     <Route path='about' component={About}/>
-    <Route path='contacts' component={Contacts}/>
+    <Route path='contacts' component={Contacts}>
+    	<Route path='address' component={Address}>
+    		<Route path='/contacts/address/another_region' component={Another_region}/>
```
Comment (denisbodnar) [inline app/src/router.js]: use spaces, not tabs

**Review by denisbodnar — DISMISSED** [review]

File: app/src/components/Another_region/Another_region.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class Another_region extends React.Component {
```
Comment (AlbaMontis) [inline app/src/components/Another_region/Another_region.js]: Use PascalCase for React components

File: app/src/components/Address/Address.js [hunk 7]
```diff
@@ -1,7 +1,14 @@
...
+    		{this.props.children}
+    	</div>
+    	)
```
Comment (AlbaMontis) [inline app/src/components/Address/Address.js]: ;

Comment (avoytovich) [reply app/src/components/Address/Address.js]: JSX not a JS!!!

**Review by denisbodnar — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

File: app/src/components/Address/Address.js [hunk 8]
```diff
@@ -1,7 +1,14 @@
...
 export default class Address extends React.Component {
   render() {
-    return <h2>You can find us in Lviv</h2>;
+    return (
+      <div>
+        <h2>You can find us in Lviv</h2>
+        <Link to="/contacts/address/another_region">Another_region</Link>
+        {this.props.children}
+      </div>
+    )
```
Comment (AlbaMontis) [inline app/src/components/Address/Address.js]: ;

**Review by denisbodnar — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

File: app/src/router.js [hunk 9]
```diff
@@ -1,12 +1,17 @@
...
+import { About, Address, AnotherRegion, Home, Contacts, Main, Maps } from './components';
 
 const Routes = (
   <Route path='/' component={Main}>
     <IndexRoute component={Home}/>
     <Route path='about' component={About}/>
-    <Route path='contacts' component={Contacts}/>
+    <Route path='contacts' component={Contacts}>
+      <Route path='address' component={Address}>
+        <Route path='/contacts/address/another_region' component={AnotherRegion}/>
```
Comment (4asof4ik) [inline app/src/router.js]: What is that path='/contacts/address/another_region'?

File: app/src/components/Address/Address.js [hunk 10]
```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { Link } from 'react-router';
 
 export default class Address extends React.Component {
   render() {
-    return <h2>You can find us in Lviv</h2>;
+    return (
+      <div>
+        <h2>You can find us in Lviv</h2>
+        <Link to="/contacts/address/another_region">Another_region</Link>
```
Comment (4asof4ik) [inline app/src/components/Address/Address.js]: ??

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

File: app/src/router.js [hunk 11]
```diff
@@ -1,12 +1,17 @@
...
+import { About, Address, AnotherRegion, Home, Contacts, Main, Maps } from './components';
 
 const Routes = (
   <Route path='/' component={Main}>
     <IndexRoute component={Home}/>
     <Route path='about' component={About}/>
-    <Route path='contacts' component={Contacts}/>
+    <Route path='contacts' component={Contacts}>
+      <Route path='address' component={Address}>
+        <Route path='/contacts/address/another' component={AnotherRegion}/>
```
Comment (4asof4ik) [inline app/src/router.js]: path should be with one name and then some params 
example: path = "/user/:id"

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by AlbaMontis — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

File: app/src/containers/index.js [hunk 12]
```diff
@@ -1 +1,7 @@
 export App from './App/App';
+export About from './About/About';
+export Address from './Address/Address';
+export Contacts from './Contacts/Contacts';
+export Home from './Home/Home';
+export Main from './Main/Main';
+export Maps from './Maps/Maps';
```
Comment (denisbodnar) [inline app/src/containers/index.js:6]: add line

**Review by denisbodnar — DISMISSED** [review]

**Review by denisbodnar — DISMISSED** [review]

**Review by yuliannakorol — DISMISSED** [review]

File: app/src/components/index.js [hunk 13]
```diff
@@ -4,4 +4,4 @@ export Address from './Address/Address';
 export Contacts from './Contacts/Contacts';
 export Home from './Home/Home';
 export Main from './Main/Main';
-export Maps from './Maps/Maps';
+export Maps from './Maps/Maps';
```
Comment (AlbaMontis) [inline app/src/components/index.js]: Add new line

**Review by denisbodnar — DISMISSED** [review]

File: app/src/components/index.js [hunk 14]
```diff
@@ -1,5 +1,5 @@
 export Hello from './Hello/Hello';
-export About from './About/About';
+export About from './../contai/About/About';
```
Comment (4asof4ik) [inline app/src/components/index.js]: './../contai.........' what is that?

File: app/src/containers/Address/Address.js [hunk 15]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router';
```
Comment (4asof4ik) [inline app/src/containers/Address/Address.js]: import but not used

**Review by 4asof4ik — CHANGES_REQUESTED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Review by yuliannakorol — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Pull request merged**